pkg/tests/package_naming_aggregate_test.sh

```diff
@@ -21,4 +21,12 @@ declare -r DATA_DIR="${TEST_SRCDIR}/rules_pkg/tests"
 for pkg in test_naming_some_value.deb test_naming_some_value.tar test_naming_some_value.zip ; do
   ls -l "${DATA_DIR}/$pkg"
 done
+
+TAR_STRUCTURE=$(tar -tf tests/test_tar_package_dir_substitution.tar)
```

Comment:
can you do this in pkg_tar_test.py instead.  Shell tests are fragile.

Reply:
Done.